tools/file_utils_test.py

```diff
@@ -0,0 +1,24 @@
+from file_utils import *
+
+
+def test_get_project_path_is_correct():
+    path = get_project_path()
+    print(path)
+
+    assert path.endswith("WomenCodingCommunity.github.io/")
+
+
+def test_get_project_path():
+    path = get_project_path()
+
+    assert "/tools" not in path
+
+
+def test_get_path_in_project():
+    assert get_path_in_project("_data").endswith("WomenCodingCommunity.github.io/_data")
+    assert get_path_in_project("tools").endswith("WomenCodingCommunity.github.io/tools")
+    assert get_path_in_project("tools/").endswith("WomenCodingCommunity.github.io/tools/")
```

Comment:
On Windows this assert is failing because it is expecting a backslash instead of slash here:
`.endswith("WomenCodingCommunity.github.io/_data")`

With use of path.join it is working fine and the test passes:
`.endswith(os.path.join("WomenCodingCommunity.github.io", "data"))`